sdk/program/src/stake/mod.rs

```diff
@@ -6,7 +6,8 @@ pub mod program {
     crate::declare_id!("Stake11111111111111111111111111111111111111");
 }
 
-/// The minimum stake amount that can be delegated, in lamports.
-/// NOTE: This is also used to calculate the minimum balance of a stake account, which is the
-/// rent exempt reserve _plus_ the minimum stake delegation.
+#[deprecated(
+    since = "1.10.6",
+    note = "This constant may be outdated, please use `solana_stake_program::get_minimum_delegation` instead"
```

Comment:
I don't think the directive here is quite right because `solana_stake_program::get_minimum_delegation` isn't public. Can we deprecate this later when you add the convenience method for doing the CPI + get_return_data?

Reply:
Yes, will do. Since the PR is already merged, I'll put up another commit to remove the deprecation.